pallets/funding/src/tests/mod.rs

```diff
@@ -15,15 +15,18 @@ use frame_support::{
 };
 use itertools::Itertools;
 use parachains_common::DAYS;
-use polimec_common::ReleaseSchedule;
+use polimec_common::{ReleaseSchedule, USD_DECIMALS};
 use polimec_common_test_utils::{generate_did_from_account, get_mock_jwt_with_cid};
 use sp_arithmetic::{traits::Zero, Percent, Perquintill};
 use sp_runtime::{BuildStorage, TokenError};
 use sp_std::{cell::RefCell, marker::PhantomData};
 use std::iter::zip;
 type MockInstantiator =
 	Instantiator<TestRuntime, <TestRuntime as crate::Config>::AllPalletsWithoutSystem, RuntimeEvent>;
-const US_DOLLAR: u128 = 1_0_000_000_000;
+const USD_UNIT: u128 = 10u128.pow(USD_DECIMALS as u32);
+const USDT_UNIT: u128 = USD_UNIT;
+const ASSET_UNIT: u128 = 10_u128.pow(10u32);
```

Comment:
Is this `ASSET_UNIT` representing PLMC? If so, would we rename it to PLMC or import PLMC const from some other place? If not, could we use the `ASSET_DECIMALS` here in the pow from line 31?

Reply:
It refers to CTs. So lets rename it to CT_UNIT, and put a different decimal count than PLMC